src/main/java/duke/Event.java

```diff
@@ -0,0 +1,70 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Event is a kind of Task.
+ * @author Lan Jingbo, Jerry
+ */
+class Event extends Task{
```

Comment:
Remember to put a space after Task. i.e. 
```suggestion
class Event extends Task {
```